tests/testthat/test-fct_relabel.R

```diff
@@ -36,3 +36,10 @@ test_that("additional arguments", {
 
   expect_identical(fct_relabel(f1, paste0, "."), factor(paste0(letters, ".")))
 })
+
+test_that("function coercion", {
```

Comment:
This should be a sentence

Reply:
The others in this file are concise. It's now a sentence, though not capitalized and with a period. If you want that (or for the others to be changed) let me know.